### DISCUSSION TITLE
Can't use in Next.js 15 and React19

### DISCUSSION THREAD

**Discussion by nextify2025** [fuma-nama/fumadocs#649] [Q&A]
Error: Cannot access DynamicSidebar.then on the server. You cannot dot into a client module from a server component. You can only pass the imported name through.
Any Suggestion ？ Thanks ！

```
    "next": "15.0.0-canary.87",
    "react": "19.0.0-rc-14a4699f-20240725",
    "react-dom": "19.0.0-rc-14a4699f-20240725"
```

**Comment by fuma-nama** [comment]
Please provide sufficient information (reproduction if possible) and submit a bug request instead.

Also Fumadocs isn't supposed to work with canary versions of Next.js. React 19 and Next.js 15 are not stable and production ready. Fumadocs won't support it until the first stable release.
